src/test/java/create_task_use_case/CreateTaskDataAccessTest.java

```diff
@@ -0,0 +1,31 @@
+package create_task_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessTask;
+import org.junit.jupiter.api.Test;
+
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CreateTaskDataAccessTest {
+    @Test
+    public void testExistsByTitleWithoutDuplicateTask(){
```

Comment:
it should. Note that since 'days' is a class attribute of WeekDataAccess and CreateTaskDataAccess has no constructor, making an instance of CreateTaskDataAccess calls the constructor for WeekDataAccess, which initializes that class attribute.

Reply:
A test case like this is arguably necessary to affirm basic functionality. If existsByTitle returned True in this case, we would be unable to create a task at all.

Reply:
I will conclude this conversation with Yes this is necessary to check adding tasks as a basic functionality and runs as intended as Daniel explains above.

Reply:
Fair. Thanks for the clarification.